testsuite/integration/pom.xml

```diff
@@ -437,6 +427,11 @@
 
     <dependencies>
         <!-- Dependencies specific to the integration tests. -->
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Which child modules use this? If it's only a few please declare it in those specifically so we know where it is needed.

As part of the big-bang work we cleaned this one up to try and reduce what's listed to things that are basically used everywhere.

Reply:
TBH, I'm not too sure. I know `testsuite/integration/web` is what failed. I can just add it there and keep testing to see which ones fail. I assume it came in via the `org.jboss.resteasy:resteasy-client` which was previously in here.

Reply:
I forgot to note this was a runtime exception. I can have a look at where better it might fit though.